packages/config/verifier-config.json

```diff
@@ -0,0 +1,83 @@
+{
+    "ledgerPluginInfo": [
+        {
+            "validatorID": "84jUisrs",
```

Comment:
@takeutak Does this file intended to be part of the published npm packages once we issue a release?